stdlib/contrib/bonitoo-io/tickscript/tickscript.flux

```diff
@@ -10,98 +11,87 @@ import "universe"
-    }))
-    |> map(fn: (r) => ({ r with id: id(r: r) }))
-    |> map(fn: (r) => ({ r with details: details(r: r) }))
-    |> _addTopic()
-    |> monitor.check(
-        crit: crit,
-        warn: warn,
-        info: info,
-        ok: ok,
-        messageFn: message,
-        data: check
-    )
+alert = (check, id=(r) => "${r._check_id}", details=(r) => "", message=(r) => "Threshold Check: ${r._check_name} is: ${r._level}", crit=(r) => false, warn=(r) => false, info=(r) => false, ok=(r) => true, topic="", tables=<-) => {
+    _addTopic = if topic != "" then
+        (tables=<-) => tables
+            |> set(key: "_topic", value: topic)
+            |> experimental.group(mode: "extend", columns: ["_topic"])
+else
+        (tables=<-) => tables
+
+    return tables
+        |> drop(fn: (column) => column =~ /_start.*/ or column =~ /_stop.*/)
+        |> map(
+            fn: (r) => ({r with
+                _check_id: check._check_id,
+                _check_name: check._check_name,
+            }),
+        )
+        |> map(fn: (r) => ({r with id: id(r: r)}))
+        |> map(fn: (r) => ({r with details: details(r: r)}))
+        |> _addTopic()
+        |> monitor.check(
+            crit: crit,
+            warn: warn,
+            info: info,
+            ok: ok,
+            messageFn: message,
+            data: check,
+        )
 }
 
 // deadman is a helper function similar to TICKscript deadman.
-deadman = (
-    check,
-    measurement, threshold=0,
-    id=(r)=>"${r._check_id}",
-    message=(r)=>"Deadman Check: ${r._check_name} is: " + (if r.dead then "dead" else "alive"),
-    topic="",
-    tables=<-) => {
-
-   // In order to detect empty stream (without tables), we concatenate input with dummy stream and count the result,
-   // because count() returns nothing for empty stream. If the input stream is empty, then dummy stream with empty
-   // table is used as input for actual threshold check in order to get 0.
-
-  _dummy = array.from(rows: [{_time: 2000-01-01T00:00:00Z, _field: "unknown", _value: 0}])
-    |> set(key: "_measurement", value: measurement)
-    |> experimental.group(columns: ["_measurement"], mode: "extend") // required by monitor.check
-    |> schema.fieldsAsCols() // input tables are expected to be pivoted already
-
-  _counts = union(tables: [_dummy, tables])
-    |> keep(columns: ["_measurement", "_time"])
-    |> duplicate(column: "_measurement", as: "__value__") // _measurement column is always present
-    |> count(column: "__value__")
-    |> findColumn(fn: (key) => key._measurement == measurement, column: "__value__")
-
-  _tables =
-    if _counts[0] == 1 then // only dummy table is in the concatenated stream
-      _dummy
-        |> drop(columns: ["unknown"])
-        |> limit(n: 0) // need empty table
+deadman = (check, measurement, threshold=0, id=(r) => "${r._check_id}", message=(r) => "Deadman Check: ${r._check_name} is: " + (if r.dead then "dead" else "alive"), topic="", tables=<-) => {
```

Comment:
So before your change, was the formatter dropping the parentheses around the if/then/else here?

Reply:
Yes, that is correct. The bracket around 'if/then/else' was getting dropped.
 
so without the current code changes formatter was returning as shown below - 
`deadman = (check, measurement, threshold=0, id=(r) => "${r._check_id}", message=(r) => "Deadman Check: ${r._check_name} is: " + if r.dead then "dead" else "alive", topic="", tables=<-) => {`